src/test/java/lambda/part2/exercise/Exercise1.java

```diff
@@ -30,35 +28,37 @@ void sameAgesCheckerUsingBiPredicate() {
         Person person2 = new Person("Дмитрий", "Гущин", 33);
         Person person3 = new Person("Илья", "Жирков", 22);
 
-        // TODO create variable sameAgesChecker: (Person, Person) -> boolean, using BiPredicate
-
-//        assertThat(sameAgesChecker.test(person1, person2), is(true));
-//        assertThat(sameAgesChecker.test(person1, person3), is(false));
-//        assertThat(sameAgesChecker.test(person2, person3), is(false));
+        //create variable sameAgesChecker: (Person, Person) -> boolean, using BiPredicate
+        BiPredicate<Person, Person> sameAgesChecker = (person1_1, person2_2) -> person1.getAge() == person2_2.getAge();
 
```

Comment:
Here is a typo, but test has passed fortunately :-))

Reply:
Thank you for recommendation. I have done it.